Scripts/FindSimilarIncidents/FindSimilarIncidents.py

```diff
@@ -85,8 +85,6 @@ def nested_dict_flatted(d, parent_key='', sep='.'):
         items = []  # type: ignore
         for k, v in d.items():
             new_key = parent_key + sep + k if parent_key else k
-            if isinstance(v, list) and len(v) > 0:
```

Comment:
why do you remove this?

Reply:
to support list comparison - now it's consider only the fist element if it's list - and it's not a good enough logic for some use-cases